### ISSUE TITLE
CKEditor field accessible name doesn't match visible label, and lacks contextual info

### ISSUE THREAD

**Issue by gcamacho079** [craftcms/ckeditor#74]
### Description
See title

### Steps to reproduce

1. Create a new custom field using the CKEditor field type, and add Default Instructions
2. Add the custom field to a field layout
3. Go to the Edit page and inspect accessibility properties: regardless of the field name, the `.ck-editor__editable` element has the accessible name “Editor editing area: main” and doesn’t reference instructions, errors, etc., as other fields do.

### Expected behavior
- The accessible name of the field should include the visible field name, preferably at the beginning. For example, if the field name is “Post Content,” the accessible name can be “Post Content, Editor editing area: main.”
- Additional information, like errors and instructions, should be referenced using `aria-describedby`

### Additional info

- Craft version: 4.4.7
- PHP version: 8.0.21
- Database driver & version: MariaDB 10.4.25
- Plugins & versions: 3.0.0

**Comment by i-just** [comment]
Hi Lupe!

I raised a PR to do this. It’s only possible to change those](https://github.com/ckeditor/ckeditor5-react/issues/100#issuecomment-507674898) once the editor is instantiated, not create it with them. LMK if this makes it worse.

**Comment by brandonkelly** [comment]
CKEditor 3.8.0 is tagged with that PR 🎉
